modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -85,6 +85,44 @@ RETURN a.name, a.year
 
 | Extension of the xref:clauses/load-csv.adoc#azure-cloud-storage[LOAD CSV] clause to allow loading CSV files from Azure Cloud Storage URIs.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+CREATE USER bob
+SET AUTH 'externalProviderName' {
+  SET ID 'userIdForExternalProvider'
+}
+SET AUTH 'native' {
+  SET PASSWORD 'password'
+  SET PASSWORD CHANGE REQUIRED
+}
+----
+| Added the ability set which link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/auth-providers.adoc[Auth Providers] apply to a user (Enterprise Edition).
+
+Administration of the `native` (username / password) auth via the new syntax is also now supported (Community Edition).
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+SHOW USERS WITH AUTH
+----
+a|
+New support for `WITH AUTH` to allow display users' Auth Providers with a separate row per user per Auth Provider.
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+[GRANT\|DENY] SET AUTH TO role
```

Comment:
might be nicer to do like we have for other privileges:
```
a|
label:functionality[]
label:new[] +
New privilege:
[source, cypher, role="noheader"]
----
ALTER DATABASE
----
a|
New privilege that allows a user to modify databases.
```
to not have this weird not-really-syntax thing

Reply:
done, thanks